assets/js/lib/test-utils/factories/users.js

```diff
@@ -12,7 +12,6 @@ export const abilityFactory = Factory.define(() => ({
 export const userFactory = Factory.define(() => ({
   id: faker.number.int(),
   username: faker.internet.userName(),
-  actions: 'Delete',
```

Comment:
Just curious why :) ?

Reply:
Because this field doesn't even exist.
I guess you added it at the beginning before knowing how it would've been.
It is `abilities` and it comes below. We should've removed this before I guess